examples/tsp-workload-resource-dsp.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: prediction.crane.io/v1alpha1
+kind: TimeSeriesPrediction
+metadata:
+  name: workload-resource-dsp
+  namespace: default
+spec:
+  predictionWindowSeconds: 3600
+  predictionMetrics:
+    - resourceIdentifier: workload-cpu
+      workloadRef:
```

Comment:
I think it would be better if we can unify workloadRef and NodeRef and only one refer is allowed for one tsp, so that the resource can be kept under each resourceIdentifier, but the workloadRef should be something like
```
  TargetRef: (for workload)
        kind: Deployment
        name: dep-1-100m-500mib
        namespace: default
```
```
TargetRef: (for node)
        kind: Node
        name: dep-1-100m-500mib
```

Reply:
> I think it would be better if we can unify workloadRef and NodeRef and only one refer is allowed for one tsp, so that the resource can be kept under each resourceIdentifier, but the workloadRef should be something like
> 
> ```
>   TargetRef: (for workload)
>         kind: Deployment
>         name: dep-1-100m-500mib
>         namespace: default
> ```
> 
> ```
> TargetRef: (for node)
>         kind: Node
>         name: dep-1-100m-500mib
> ```

OK, I will move the TargetRef to the tsp scope, and each tsp associate just only one TargetRef